src/components/modals/bulkTopicsModal.tsx

```
@@ -0,0 +1,139 @@
+              </Modal.Description>
+            </div>
+
+            <div class="flex flex-col gap-2 mt-2 text-sm p-2 rounded-md overflow-y-auto">
+              <div class="flex justify-between items-center py-2 border-b border-gray-300 dark:border-gray-600">
+                <div class="font-semibold dark:text-white">Current Tags</div>
+                <div class="font-semibold dark:text-white">Remove?</div>
+              </div>
+              {selectedRepos.value
+                .reduce((allTopics, repoName) => {
+                  const repoTopics = topicsMap?.[repoName] || [];
+                  return [...new Set([...allTopics, ...repoTopics])];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)